docs/source/user/aerodyn/introduction.rst

```diff
@@ -68,10 +69,10 @@ driver code, without aero-elastic coupling.
 
 AeroDyn consists of four submodels: (1) rotor wake/induction, (2) blade
 airfoil aerodynamics, (3) tower influence on the fluid local to the
-blade nodes, and (4) tower drag. Nacelle, hub, and tail-vane fluid
-influence and loading, aeroacoustics, and wake and array effects between
-multiple turbines in a wind plant, are not yet available in AeroDyn v15
-and newer.
+blade nodes, and (4) tower drag. Nacelle, hub, and tail-vane fluid 
+influence and loading (with the exception of nacelle and hub buoyant
+loads), aeroacoustics, and wake and array effects between multiple 
+turbines in a wind plant, are not yet available in AeroDyn v15 and newer.
```

Comment:
This statement seems a bit out of date.  Aeroacoustics are now available (except for MHK turbines).  I'd suggest adding "buoyancy in the blades, hub, nacelle, and tower (for MHK turbines)" as item (5) in the list.

Reply:
The for the comment. Just to clarify, does the following phrasing sound good?

"AeroDyn consists of five submodels: (1) rotor wake/induction, (2) blade airfoil aerodynamics, (3) tower influence on the fluid local to the blade nodes, (4) tower drag, and (5) buoyancy on the blades, hub, nacelle, and tower (for MHK turbines). Nacelle, hub, and tail-vane fluid influence and loading (with the exception of nacelle and hub buoyant loads) and wake and array effects between multiple turbines in a wind plant are not yet available in AeroDyn v15 and newer."

Reply:
Yes, but I'd suggest adding "aeroacoustics" as bullet (5), making buoyancy bullet (6).

Reply:
I've made these changes in the updated doc.